fastlane/Fastfile.swift

```diff
@@ -176,6 +176,24 @@ class Fastfile: LaneFile {
         Match.syncCodeSigning(type: .adHoc, environment: .staging, isForce: true)
     }
 
+    func addDevicesGenerateProfilesLane() {
```

Comment:
Shall we mention `Staging` since that is the env we only sync for now?
```suggestion
    func addDevicesGenerateProfilesForStagingLane() {
```

Reply:
We may have the dev env as SM did, so it's better to keep a generic name here